### PR TITLE
Alternative EC documentation

### PR DESCRIPTION
**Pull request by lucassaldanha**
## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [X] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [X] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [X] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [x] you fixed all the issues raised by the tests, if any.
- [x] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Added documentation on how to use alternative elliptic curves in Besu.

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

### CODE REVIEW

File: docs/Reference/Alternative-EC-Curves.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    "config": {
+      "ecCurve": "secp256k1",
+    [...]
+  },
+  [...]
+}
+```
+
+!!! attention
+    All nodes in the network **MUST** use the same elliptic curve.
```
Comment (NicolasMassart) [inline docs/Reference/Alternative-EC-Curves.md:26]: Could you say more about the consequences? Will it corrupt something or just break the consensus? What if it happens? does it break something and can we fix if we have a node joining with the wrong EC scheme?

File: docs/Reference/Alternative-EC-Curves.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+The following Elliptic Curves are supported by Besu:
```
Comment (NicolasMassart) [inline docs/Reference/Alternative-EC-Curves.md:28]: Could provide some hints about the use case for which we should use one or the other?

Comment (lucassaldanha) [reply docs/Reference/Alternative-EC-Curves.md:28]: At the moment the only reason is that secp256r1 is NIST approved and sepc256k1 is not. But this is just the first step towards a compliant node so I don't really think we should mention it in the docs for now.

I'd say that the users interested in using alternative keys will know why they need to do so.

**Review by bgravenorst — APPROVED** [review]
LGTM

**Pull request merged**